WordPress/src/main/java/org/wordpress/android/ui/posts/PostsListFragment.java

```diff
@@ -552,8 +552,7 @@ private void publishPost(final PostModel post) {
         PostUtils.updatePublishDateIfShouldBePublishedImmediately(post);
         post.setStatus(PostStatus.PUBLISHED.toString());
 
-        PostUploadService.addPostToUpload(post);
-        getActivity().startService(new Intent(getActivity(), PostUploadService.class));
+        UploadService.addPostToUpload(getActivity(), post);
```

Comment:
The inconsistency between how we added things to the two services was really starting to get to me 😉